CONTRIBUTING.md

```diff
@@ -13,14 +14,6 @@ Prepare context, which was wrong.
 
 Fixes #113
 ```
+
 Please apply the same logic for Pull Requests, start with the package name, followed by a colon and a description of the change, just like
 the official [Go language](https://github.com/golang/go/pulls).
-
-
-### Releasing a new version
```

Comment:
Even if we are gonna automate releases, it'd still be nice to explain how it works and how to trigger it.

Reply:
I see now that you've added a dedicated file for it below :)